packages/platform-core/src/usage-data/cli_telemetry_data.ts

```diff
@@ -0,0 +1,70 @@
+// Core Identifiers
+export type CoreIdentifiersDetails = {
+  sessionUuid: string;
+  timestamp: string;
+  localProjectId: string;
+  accountId?: string;
+  payloadVersion: string;
+  awsRegion: string;
+};
+
+// Command Details
+export type CommandDetails = {
+  success: boolean;
+  command: {
+    path: string[];
+    parameters: string[];
+  };
+  executionTime: number;
```

Comment:
I would move `executionTime` under `Latency` and call it `total` or something like that.

Reply:
Addressed it